bread/static/js/main.js

```diff
@@ -128,6 +128,13 @@ function formset_add(form_prefix, list_container) {
     formcount.value = parseInt(formcount.value) + 1;
     update_add_button(form_prefix);
     updateMultiselect(container_elem);
+
+    htmx.process(container_elem);
+}
+
+function set_value(selector, value) {
+    let element = $(selector)
+    element.value = value
 }
```

Comment:
Not sure why this is needed, see comment in basxconnect PR.

Reply:
you're right, it is better to inline that one